### PR TITLE
Added Rails 7 support

### PR DESCRIPTION
**Pull request by skunkworker**
Adds Rails 7.0 support.
Fixes some ruby 3.1 syntax bugs

### CODE REVIEW

**Comment by skunkworker** [comment]
`:to_fs` replaces `:to_s` on the date objects, see https://stackoverflow.com/questions/71177165/rails-ignores-the-default-date-format-after-upgrading-from-6-1-to-7-0

**Comment by skunkworker** [comment]
@liveh2o could you take a look whenever convenient?

File: lib/active_remote/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module ActiveRemote
-  VERSION = "6.1.1"
+  VERSION = "7.0.0"
```
Comment (liveh2o) [inline lib/active_remote/version.rb:2]: I prefer to bump the version while releasing to more easily group changes together.

**Review by liveh2o — APPROVED** [review]

**Comment by liveh2o** [comment]
Released in [v7.0.0](https://github.com/liveh2o/active_remote/releases/tag/v7.0.0).

**Pull request merged**